apps/marketplace/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styles from './PageHeader.scss'
+
+class PageHeader extends React.PureComponent {
```

Comment:
Why `PureComponent` as opposed to a stateless component? I don't think we've used a `PureComponent` yet.